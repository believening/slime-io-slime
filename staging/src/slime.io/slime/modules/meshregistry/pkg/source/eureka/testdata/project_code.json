[
    {
      "name": "ApplicationA",
      "instance": [
        {
          "instanceId": "appA-instance-1",
          "hostName": "192.168.2.10",
          "ipAddr": "192.168.2.10",
          "status": "UP",
          "port": {
            "$": 8000,
            "@enabled": "true"
          },
          "securePort": {
            "$": 8443,
            "@enabled": "false"
          },
          "app": "ApplicationA",
          "metadata": {
            "version": "2.1.0",
            "zone": "eu-west-1a",
            "projectCode": "project1"
          }
        },
        {
          "instanceId": "appA-instance-2",
          "hostName": "192.168.2.11",
          "ipAddr": "192.168.2.11",
          "status": "UP",
          "port": {
            "$": 8001,
            "@enabled": "true"
          },
          "securePort": {
            "$": 8444,
            "@enabled": "false"
          },
          "app": "ApplicationA",
          "metadata": {
            "version": "2.1.0",
            "zone": "eu-west-1b",
            "projectCode": "project2"
          }
        }
      ]
    },
    {
      "name": "ApplicationB",
      "instance": [
        {
          "instanceId": "appB-instance-1",
          "hostName": "192.168.3.10",
          "ipAddr": "192.168.3.10",
          "status": "UP",
          "port": {
            "$": 9000,
            "@enabled": "true"
          },
          "securePort": {
            "$": 9443,
            "@enabled": "false"
          },
          "app": "ApplicationB",
          "metadata": {
            "version": "1.2.0",
            "zone": "ap-southeast-1a",
            "projectCode": "project3"
          }
        }
      ]
    }
  ]