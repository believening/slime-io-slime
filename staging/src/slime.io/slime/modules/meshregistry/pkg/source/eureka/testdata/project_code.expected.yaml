---
kind: ServiceEntry
apiVersion: networking.istio.io/v1alpha3
metadata:
  name: applicationa.nsf.project1.eureka
  namespace: eureka
  labels:
    registry: eureka
  annotations: {}
spec:
  hosts:
    - applicationa.nsf.project1.eureka
  ports:
    - number: 80
      protocol: HTTP
      name: http-80
  resolution: STATIC
  endpoints:
    - address: 192.168.2.10
      ports:
        http-80: 8000
      labels:
        projectCode: project1
        version: 2.1.0
        zone: eu-west-1a
---
kind: ServiceEntry
apiVersion: networking.istio.io/v1alpha3
metadata:
  name: applicationa.nsf.project2.eureka
  namespace: eureka
  labels:
    registry: eureka
  annotations: {}
spec:
  hosts:
    - applicationa.nsf.project2.eureka
  ports:
    - number: 80
      protocol: HTTP
      name: http-80
  resolution: STATIC
  endpoints:
    - address: 192.168.2.11
      ports:
        http-80: 8001
      labels:
        projectCode: project2
        version: 2.1.0
        zone: eu-west-1b
---
kind: ServiceEntry
apiVersion: networking.istio.io/v1alpha3
metadata:
  name: applicationb.nsf.project3.eureka
  namespace: eureka
  labels:
    registry: eureka
  annotations: {}
spec:
  hosts:
    - applicationb.nsf.project3.eureka
  ports:
    - number: 80
      protocol: HTTP
      name: http-80
  resolution: STATIC
  endpoints:
    - address: 192.168.3.10
      ports:
        http-80: 9000
      labels:
        projectCode: project3
        version: 1.2.0
        zone: ap-southeast-1a